doc/docs/master/getting_started/local_installation.md

```diff
@@ -166,6 +166,16 @@ deploy Pachyderm on your local cluster by following these steps:
    $ helm install pachd pach/pachyderm --set deployTarget=LOCAL
    ```
 
+* To install pachyderm with console and authentication set up, run the following helm installation:
```

Comment:
I hope this will be a really helpful and easily discovered snippet for how to auto-magically deploy pach + console locally